src/layout/footer/index.module.scss

```
@@ -0,0 +1,42 @@
+.footer {
+    background-color: #eee;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: flex-start;
+    padding: 50px 0;
+    font-size: 16px;
+    @media screen and (max-width: 600px) {
+        flex-direction: column-reverse;
+        & > div {
+            text-align: center;
+        }
+    }
+    & > div {
+        margin: auto;
+        padding: 10px;
+    }
+    & h4 {
```

Can't you just get rid of `&`
```suggestion
    h4 {
```